jekyll/_cci2/orbs-user-config.md

```diff
@@ -39,17 +39,18 @@ The example below demonstrates how you can invoke the `hello-build` orb in the `
 ```yaml
 version: 2.1
 orbs:
-    hello: circleci/hello-build@0.0.5
+  hello: circleci/hello-build@0.0.5
 
 workflows:
-    "Hello Workflow":
-        jobs:
-          - hello/hello-build
+  "Hello Workflow":
+    jobs:
+      - hello/hello-build
 ```
 
 **Note:** If your project was added to CircleCI prior to 2.1, you must enable pipelines to use the `orbs` key.
```

Comment:
Here too?

Reply:
Or maybe you're just replacing it when it's a step not just a note.

Reply:
Thanks for pointing this one out. I actually think we should axe this line because it's implied that we're using 2.1 as it's requested at the top of this file (thus enabling it for orbs as well).